From 626a80b976f94331710a1e9c3e9c9b95558ec557 Mon Sep 17 00:00:00 2001
From: Varun Kumar Reddy Yaparla <vyaparla@habana.ai>
Date: Mon, 27 Dec 2021 18:38:24 +0200
Subject: [PATCH] [SW-67662] gaudi2: arc: increase stack size

This change increases gaudi2 arc preboot stack size from
2KB to 4KB. This change is to avoid stack overflow issue
when signature validation flow is enabled

Change-Id: Iab93d767e7060e72906e7ab73652f8b1893228e6
---

diff --git a/arch/gaudi2/arc/preboot.lds.S b/arch/gaudi2/arc/preboot.lds.S
index a467ebd..1bc858f 100644
--- a/arch/gaudi2/arc/preboot.lds.S
+++ b/arch/gaudi2/arc/preboot.lds.S
@@ -31,7 +31,7 @@
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
-		. = . + 2048 ;
+		. = . + 4096 ;
 		__stack_top = . ;
 	} > .sram
 
